string assignmentId
string hitId
string workerId
string turkSubmitTo
---
string topic
